.pre-commit-config.yaml

```diff
@@ -83,3 +83,8 @@ repos:
     rev: '1.5.0'
     hooks:
     -   id: poetry-check
+    -   id: poetry-lock
+        name: validate poetry lock
+        files: '^(pyproject.toml)|(poetry.lock)$'
```

Comment:
just need to add the poetry.lock file to the release files

Reply:
ok, added lock and toml to common release files - both are needed for the check

Reply:
is this files line still needed?

Reply:
they are not i can delete them

Reply:
btw the build_release job handles files required by those checks differently, it copies them over, instead of having them in release files:
```
    - name: pre-commit
      run: |
        cd $GITHUB_WORKSPACE
        cp .pre-commit-config.yaml $STRIPPED_DIR
        cp .pylintrc $STRIPPED_DIR
        cp mypy.ini $STRIPPED_DIR
        cd $STRIPPED_DIR
        ${{ env.RUN }} "pre-commit run --all"
```
should i align?

Reply:
yeah let's do that, good catch